### PR TITLE
Styled components

### PR DESCRIPTION
**Pull request by lauravikanis**
Adding styled-components, global styles and variables. 🎨

### CODE REVIEW

File: client/src/App.js [hunk 1]
```diff
@@ -1,17 +1,28 @@
...
+import GlobalStyle from "./GlobalStyles";
+import styled from "styled-components";
+
+const Main = styled.main`
+  padding: 1rem;
+  max-width: 375px;
+  display: flex;
+  flex-direction: column;
+  margin: auto;
+`;
```
Comment (mariereese) [inline client/src/App.js:13]: I would move the styles to the bottom of this file to make it more concise

File: client/src/components/Header/Header.js [hunk 2]
```diff
@@ -2,7 +2,7 @@ import React from "react";
 import PropTypes from "prop-types";
 
 import Button from "../Button";
-import "./header.css";
+// import "./header.css";
```
Comment (mariereese) [inline client/src/components/Header/Header.js:6]: you can delete the comments before merging

**Review by mariereese — COMMENTED** [review]
Looks good :)

File: client/src/App.js (hunk 1 above)
Comment (lauravikanis) [reply client/src/App.js:13]: In all the projects I've done before, they were placed on top of the file. 🤷‍♀️

**Review by kschmitz123 — APPROVED** [review]
lgtm 👍

**Pull request merged**